src/main/scala/com/timushev/sbt/updates/UpdatesPlugin.scala

```diff
@@ -13,10 +13,11 @@ object UpdatesPlugin extends AutoPlugin {
   override val projectSettings = Seq(
     dependencyUpdatesReportFile := target.value / "dependency-updates.txt",
     dependencyUpdatesExclusions := DependencyFilter.fnToModuleFilter(_ => false),
+    dependencyUpdatesFilter := DependencyFilter.fnToModuleFilter(_ => true),
```

Comment:
I didn't change the type here to a DependencyFilter since it seemed a bit more general than the purpose of the plugin in general and complicated the Reporter check since there are other facets that a DependencyFilter can cover, since things like ConfigurationFilter extend it.